assets/sass/_grid-settings.scss

```diff
@@ -24,3 +24,6 @@ $large-screen: em(860);
 
 $medium-screen-up: new-breakpoint(min-width $medium-screen 4);
 $large-screen-up: new-breakpoint(min-width $large-screen 8);
+
+$mobile: 1140px;
+$go-mobile: new-breakpoint(max-width $mobile $grid-columns);
```

Comment:
@nucro  What this variable means? can you make your variables more meaningful.

Thank you